<launch>
    <node pkg="rosbag" type="play" name="rosbag" output="screen" args="--loop $(find sonar_mapping)/rosbag/static_sonar.bag"/>
    <node pkg="sonar_mapping" type="filtering.py" name="get_data" output="screen"/>

    <include file="$(find point_cloud_converter)/launch/point_cloud_converter.launch" />

    <node pkg="octomap_server" type="octomap_server_node" name="octomap_server">
    <param name="resolution" value="0.05" />
    
    <!-- fixed map frame (set to 'map' if SLAM or localization running!) -->
    <param name="frame_id" type="string" value="sonar" />
    
    <!-- maximum range to integrate (speedup!) -->
    <param name="sensor_model/max_range" value="5.0" />
    
    <!-- data source to integrate (PointCloud2) -->
    <remap from="cloud_in" to="/tritech_micron/pointcloud2/filttered_scan" />

</node>
</launch>
